llvm/lib/Target/LoongArch/MCTargetDesc/LoongArchMCTargetDesc.cpp

```diff
@@ -61,7 +62,14 @@ createLoongArchMCSubtargetInfo(const Triple &TT, StringRef CPU, StringRef FS) {
 static MCAsmInfo *createLoongArchMCAsmInfo(const MCRegisterInfo &MRI,
                                            const Triple &TT,
                                            const MCTargetOptions &Options) {
-  MCAsmInfo *MAI = new LoongArchMCAsmInfo(TT);
+  MCAsmInfo *MAI;
+
```

Comment:
delete blank line

we don't typically insert a blank line after variable declaration.